### PR TITLE
Some fixes and changes.

### PR DESCRIPTION
**Pull request by Lysandr0**

### CODE REVIEW

**Comment by Lysandr0** [comment]
Made more clean-up work and ironed out some stuff over my fork for personal purposes, but I could merge it in this one if desired (assuming that there's no major problem with it). :p

File: NCPCore/src/main/java/fr/neatmonster/nocheatplus/checks/moving/player/SurvivalFly.java [hunk 1]
```diff
@@ -1695,6 +1684,9 @@ else if (Bridge1_13.isRiptiding(player) || (data.timeRiptiding + 3000 > now)) {
                 else if ((totalVDistViolation < 0.8 && data.liftOffEnvelope == LiftOffEnvelope.LIMIT_LIQUID)) {
                     // Ignore water logged blocks 
                 }
+                else if (Bridge1_9.isGlidingWithElytra(player)){
+                    // Don't care about players gliding
+                }
```
Comment (xaw3ep) [inline NCPCore/src/main/java/fr/neatmonster/nocheatplus/checks/moving/player/SurvivalFly.java:1689]: It won't work as the method returns true, they would stay on CreativeFly instead. If it was that simple, I wouldn't have to add tons of fake velocities thing.

**Comment by xaw3ep** [comment]
> Made more clean-up work and ironed out some stuff over my fork for personal purposes, but I could merge it in this one if desired (assuming that there's no major problem with it). :p

Just a quick look through, and I actually love it. merge it anyway!

File: NCPCore/src/main/java/fr/neatmonster/nocheatplus/checks/moving/player/SurvivalFly.java (hunk 1 above)
Comment (Lysandr0) [reply NCPCore/src/main/java/fr/neatmonster/nocheatplus/checks/moving/player/SurvivalFly.java:1689]: Yeah i just thought about it. I think your fix with data#keepFrictionTick fixed it anyway, will remove this one.

**Comment by Lysandr0** [comment]
Will create a different PR for it. Just to keep things in order.

**Comment by xaw3ep** [comment]
Maybe move against check fix too, there almost nothing left here.

**Comment by Lysandr0** [comment]
Done, will close this one then. :p

**Pull request closed without merging**